src/app/organizations/collection/collection.component.html

```diff
@@ -23,7 +23,7 @@
     <div class="container" *ngIf="(collection$ | async) as collection" fxLayout="column">
       <mat-card fxFlex class="my-3 alert alert-info" *ngIf="organization?.status === pendingEnum">
         <mat-icon>info</mat-icon> This collection is part of an organization that is pending approval by a Dockstore curator.
-        You can view your pending organization requests on the requests tab of the <a [routerLink]="('/accounts')" style="text-decoration: underline">accounts page</a>.
+        Relevant pending organization requests are available on the requests tab of the <a [routerLink]="('/accounts')" style="text-decoration: underline">accounts page</a>.
```

Comment:
This doesn't seem like an improvement (seems more passive?)

Reply:
Would saying 'You can view ~your~ pending organization requests on the requests tab of the accounts page' be better?

Reply:
Maybe, mainly I think just saying "Relevant  pending organization requests ... " is unclear since a user probably doesn't know what relevant means or might have a different intuition of what relevant means.  

So something more direct like, "As a curator, you can view and approve requests people have made to create organizations ..."

Reply:
Ok so have different text for curators and for members?

Reply:
If its not too much of a pain